packages/application-components/src/components/main-pages/internals/main-page.styles.ts

```diff
@@ -15,5 +15,5 @@ export const MainPageContent = styled.div`
   flex: 1;
   flex-basis: 0;
   overflow: auto;
-  padding: ${appKitDesignTokens.marginForPageContent};
+  margin: ${appKitDesignTokens.marginForPageContent};
```

Comment:
Thanks, I forgot about this.

Let's leave a comment then to avoid these changes in the future.

Reply:
Sure 👍 done cac73636bdbfb732f83a99d3f7adf6440e1433f6

Reply:
Maybe as a follow-up we can investigate why this change broke the `DataTable` styles.

Reply:
See #3117 for the explanation